AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/handler/FragmentArgHandler.java

```diff
@@ -113,5 +113,12 @@ private void createBuilderInjectionMethod(Element element, EFragmentHolder holde
 		JVar arg = method.param(paramClass, fieldName);
 		method.body().invoke(builderArgsField, bundleHelper.getMethodNameToSave()).arg(argKeyStaticField).arg(arg);
 		method.body()._return(_this());
+
+		String docComment = processingEnv.getElementUtils().getDocComment(element);
+		if (docComment != null) {
+			method.javadoc().append(docComment.replaceAll("\r", "").trim());
```

Comment:
I would extract this whole block into one method:

if (docComment != null) {   method.javadoc().append(codeModelHelper.
trimDocComment(docComment));  }